package.json

```diff
@@ -4348,7 +4348,7 @@
         "@aws-sdk/property-provider": "3.46.0",
         "@aws-sdk/smithy-client": "^3.46.0",
         "@aws-sdk/util-arn-parser": "^3.46.0",
-        "@aws/mynah-ui-chat": "npm:@aws/mynah-ui@3.2.10",
+        "@aws/mynah-ui-chat": "npm:@aws/mynah-ui@3.3.2",
```

Comment:
why does this have a `npm:` prefix?

Reply:
As i've seen it is just to rename the package which also not necessary since we don't have the search version here anymore. I'll also update that but since it will affect the files which uses mynah-ui it is better to keep it for another PR.